src/Components/IndexScene/CustomVariableValueSelectors.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  ControlsLayout,
+  SceneComponentProps,
+  sceneGraph,
+  SceneObjectBase,
+  SceneObjectState,
+  VariableValueSelectWrapper,
+} from '@grafana/scenes';
+import React from 'react';
+
+export interface VariableValueSelectorsState extends SceneObjectState {
+  layout?: ControlsLayout;
+  includeNames?: string[];
+  excludeNames?: string[];
+}
+
+export class CustomVariableValueSelectors extends SceneObjectBase<VariableValueSelectorsState> {
+  public static Component = CustomVariableValueSelectorsRenderer;
+}
+
+function CustomVariableValueSelectorsRenderer({ model }: SceneComponentProps<CustomVariableValueSelectors>) {
+  const variablesSetState = sceneGraph.getVariables(model)!.useState();
```

Comment:
I'm always a bit scared seeing `!`. Can we fail gracefully if not defined, e.g. just setting `variables` to an empty error.

Reply:
I'll take a look, this was lifted directly from scenes though: https://github.com/grafana/scenes/blob/main/packages/scenes/src/variables/components/VariableValueSelectors.tsx#L22

Reply:
Nevermind, seems like it wasn't needed. Removed!